package.json

```diff
@@ -168,19 +168,19 @@
     "flow-bin": "^0.53.1",
     "font-awesome-webpack": "~0.0.4",
     "git-rev-sync": "^1.8.0",
-    "gulp": "^3.9.0",
+    "gulp": "^4.0.0",
```

Comment:
We can remove gulp. I didn't find usage anywhere

cc @bsclifton

Reply:
what about other tools like Flow?

Reply:
@Slava i started setting up flow but it never really got used. i  don't think anything runs flow check anymore

Reply:
yup- we can remove gulp. We do use it elsewhere (ex: when packaging our version of PDF.js), but not in browser-laptop

Reply:
++ on removing flow

Reply:
the subsequent commit removes both (as well as references to flow plugins)